build.sbt

```diff
@@ -42,6 +42,7 @@ def conditionalDependsOn(prj: Project): Project = {
 }
 
 lazy val example = conditionalDependsOn(project in file("."))
+  .dependsOn(boom)
```

Comment:
You should add sifive-blocks to the build.sbt and make the top-level example project depend on all generator packages IMO.